vendor.mod

```diff
@@ -4,15 +4,17 @@ module github.com/docker/cli
 // There is no 'go.mod' file, as that would imply opting in for all the rules
 // around SemVer, which this repo cannot abide by as it uses CalVer.
 
-go 1.19
+go 1.21
+
+toolchain go1.21.8
```

Comment:
Hmm.. looks like the containerd update forces go 1.21 to be set here,  and also inserts a `toolchain`. Not sure if we want/need that? Looks like it's possible to manually remove it (and won't add it back)